packages/techdocs-common/package.json

```diff
@@ -44,6 +44,7 @@
     "@backstage/errors": "^0.1.1",
     "@backstage/integration": "^0.6.0",
     "@google-cloud/storage": "^5.6.0",
+    "@trendyol-js/openstack-swift-sdk": "^0.0.4",
```

Comment:
Don't forget to remove the `pkgcloud` dependency.

Reply:
yep, we want to see it disappear from the lockfile! :)